pcsx2/gui/GlobalCommands.cpp

```diff
@@ -70,10 +70,13 @@ namespace Implementations
 		g_Conf->EmuOptions.GS.FrameSkipEnable = !g_Conf->EmuOptions.GS.FrameSkipEnable;
 		SetGSConfig().FrameSkipEnable = g_Conf->EmuOptions.GS.FrameSkipEnable;
 
-		if( EmuConfig.GS.FrameSkipEnable )
+		if( EmuConfig.GS.FrameSkipEnable ) {
+			OSDlog( wxColor(L"RED"), "(FrameSkipping) Enabled : FrameDraws=%d, FrameSkips=%d", g_Conf->EmuOptions.GS.FramesToDraw, g_Conf->EmuOptions.GS.FramesToSkip );
```

Comment:
I feel the same way, but obviously you wouldn't want everything that is logged to to console to be displayed on screen.  Which would mean a change to the console interface for an OSD flag.  I think it would be a great change in the future, but to keep this change set small and get it into the mainline soon I would leave it as is.  If you feel strongly that this needs to change now, I can look into altering the console api.